modules/ca-bundle-understanding.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * security/certificates/updating-ca-bundle.adoc
+
+:_content-type: SNIPPET
+[id="ca-bundle-understanding_{context}"]
+= Understanding the CA Bundle certificate
+
+Proxy certificates allow users to specify one or more custom certificate authority (CA) certificates used by platform components when making egress connections.
```

Comment:
This sentence confused me a bit with certificate mentioned a few times. What do you think of removing that last use of certificates?
```suggestion
Proxy certificates allow users to specify one or more custom certificate authority (CA) used by platform components when making egress connections.
```

Reply:
Good catch! Change committed; pushing.